### ISSUE TITLE
More sound events and themes

### ISSUE THREAD

**Issue by devinprater** [nvaccess/nvda#9738]
### Is your feature request related to a problem? Please describe.
Outspoken NVDA addon deprecated

### Describe the solution you'd like
Control types should be able to have sounds assigned to them, so that if a user moves focus to a button or edit control, NVDA simply replaces the spoken description of the control with the designated sound. Another option would be to get the coords of the control, and optionally use [Windows Sonic for Headphones](https://docs.microsoft.com/en-us/windows/desktop/coreaudio/spatial-sound) to show where the control is onscreen, allowing the user to form spacial awareness of the applications they use.

### Describe alternatives you've considered
The only real alternative is the Unspoken NVDA addon, which is no longer maintained.

### Additional context
Perhaps this could start as part of the Windows 10 Essentials addon, since Windows Sonic for Headphones is only available on Windows 10, but I'd definitely urge that this be in source, with the Spacial Sound settings hidden for non-Windows-10 users

Also, I'd recommend that a sound themes manager be implemented as well, to further allow the community to customize the sounds. Perhaps, even, the Themes could be NVDA addons themselves, drawing more attention to the NVDA community addons page.

**Comment by josephsl** [comment]
Hi, location awareness work is #2559 and I have worked on it. The other ones are part of Audio Themes which is incompatible although someone is trying to make it work in more recent NVDA releases. Thanks.

**Comment by Qchristensen** [comment]
Re the last suggestion in the original comment about a sound themes interface.  I have created a separate issue for that - #16695 - and as Joseph noted, #2559 covers audio coordinates for object navigation.  Let's then use this issue to explore earcons for control types.

In NVDA's document formatting settings, near the bottom under "Elements" there is a list of options which can be toggled as to whether they are identified by speech currently:
Headings
Links
Graphics
Lists
Blockquotes
Groupings
Landmarks and regions
Articles
Frames
Figures and Captions
Clickable


Would that be a good list to work on for this feature?  Or would it be worth starting with a different list of elements (eg edit boxes aren't in this list)

**Comment by cary-rowen** [comment]
Hi @seanbudd 

Do you recommend implementing this Feature request step by step?
For example, first implement the use of sound replace link role.

**Comment by seanbudd** [comment]
@cary-rowen probably not, no. It would be important to have a general multipurpose design from the start

**Comment by mltony** [comment]
FWIW I have revived the functionality of Unspoken add-on. Just released "Earcons and Speech Rules" add-on - available in add-on store.
